data/json/items/ammo/10mm.json

```diff
@@ -26,7 +26,7 @@
     "id": "bp_10mm_fmj",
     "copy-from": "10mm_fmj",
     "type": "AMMO",
-    "name": "10mm Auto FMJ blackpowder",
+    "name": "10mm Auto FMJ (reloaded, blackpowder)",
```

Comment:
IMHO "blackpowder" already suggests the item is "reloaded".

Reply:
To add to this, there are two further layers of redundancy:
- The fact that blackpowder ammo is reloaded is stated in its item description.
- Blackpowder ammo doesn't spawn afaik, so the player will always have crafted it, meaning they know quite well that it is reloaded.